packages/content-fetch/youtube-handler.js

```diff
@@ -36,11 +37,13 @@ exports.youtubeHandler = {
 
     const oembedUrl = `https://www.youtube.com/oembed?format=json&url=` + encodeURIComponent(`https://www.youtube.com/watch?v=${videoId}`)
     const oembed = (await axios.get(oembedUrl.toString())).data;
-    const title = oembed.title;
+    // escape html entities in title
+    const title = _.escape(oembed.title);
```

Comment:
So if the input was `This "is quoted"` would the output here be `This \"is quoted\"`?

Reply:
Oh and then we unescape when we set the title in the database?

Reply:
Yes, it will become `This \"is quoted\"` in the database